cobalt/android/apk/app/src/org/chromium/content_shell/Shell.java

```diff
@@ -8,13 +8,14 @@
 import android.graphics.Rect;
```

Comment:
My key concern is changes to chromium/content_shell code

But getting a bug root caused with this extra logging is far more important at the moment than preserving unmodified Chromium code.

So i'm okay to get this merged and later reverted at least in these files once we have had nightly builds to test with.

Reply:
Disregard, this is in cobalt dir already.